.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.18.0
+  shared: getoutreach/shared@dev:first
```

Comment:
Should this be dev:first?

Reply:
It wants it to be :( I'll manually set it to `2.19.2` for now